src/Wallabag/CoreBundle/Repository/EntryRepository.php

```diff
@@ -128,7 +128,7 @@ public function findEntries($userId, $isArchived = null, $isStarred = null, $sor
             $qb->andWhere('e.isStarred =:isStarred')->setParameter('isStarred', (bool) $isStarred);
         }
 
-        if ($since >= 0) {
+        if ($since > 0) {
```

Comment:
Because when `$since` is 0, `new \DateTime(date('Y-m-d H:i:s', $since))` with generate a date from January 1970. And on every request to the api, you will always add this condition to the query because 0 is the default value of `$since`.